diff --git a/rom/ElkWifi.asm b/rom/ElkWifi.asm
--- a/rom/ElkWifi.asm
+++ b/rom/ElkWifi.asm
@@ -22 +22 @@
-                    equb &0B                    \ 1MHzWifi ROM release 0.1.11
+                    equb &0C                    \ 1MHzWifi ROM release 0.1.12
@@ -25 +25 @@
-.romversion         equs "0.1.11"               \ project test version
+.romversion         equs "0.1.12"               \ project test version
@@ -77 +77 @@
-                    equs "1MHzWifi 0.1.11",&EA
+                    equs "1MHzWifi 0.1.12",&EA
