diff --git a/rom/ElkWifi.asm b/rom/ElkWifi.asm
--- a/rom/ElkWifi.asm
+++ b/rom/ElkWifi.asm
@@ -24 +24 @@
-                    equb &01                    \ 1MHzWifi ROM release 0.1.9
+                    equb &0A                    \ 1MHzWifi ROM release 0.1.10
@@ -28 +28 @@
-.romversion         equs "0.1.9"                \ project test version
+.romversion         equs "0.1.10"               \ project test version
@@ -77 +77 @@
-                    equs "1MHzWifi 0.1.9",&EA
+                    equs "1MHzWifi 0.1.10",&EA
