diff --git a/rom/ElkWifi.asm b/rom/ElkWifi.asm
--- a/rom/ElkWifi.asm
+++ b/rom/ElkWifi.asm
@@ -161,5 +161,9 @@
 .commandtable       equs "WGET"
                     equb >pi_wget_cmd, <pi_wget_cmd
+                    equs "QUPRUN"
+                    equb >uef_run_cmd, <uef_run_cmd
+                    equs "UEF"
+                    equb >uef_cmd, <uef_cmd
                     equs "MENUSRC"
                     equb >menusrc_cmd, <menusrc_cmd
                     equs "MENU"
@@ -236,6 +242,7 @@
                     equs " TIME      Print current time",&0D
                     equs " VERSION   Print firmware version",&0D
                     equs " WGET      Get a file from a webserver",&0D
+                    equs " UEF       Load local UEF file",&0D
                     equs " WICFS     Enable WiFi CFS",&0D
                     equs " WIFI      WiFi control ON|OFF|HR|SR",&0D
                     nop
@@ -290,6 +297,7 @@
 include "wicfs.asm"
 include "wget.asm"
 include "net_wget.asm"
+include "uef.asm"
 include "menu.asm"
 include "menusrc.asm"
 include "ping.asm"
